packages/harpguru-core/src/components/harp-cell-accessible/harp-cell-accessible.tsx

```diff
@@ -60,19 +86,23 @@ const areEqual = (
   const equalStyle = prevProps.baseStyles.width === nextProps.baseStyles.width
   const equalDegree = prevProps.degreeId === nextProps.degreeId
   const equalPitch = prevProps.pitchId === nextProps.pitchId
-  const equalActive = prevProps.isActive === nextProps.isActive
+  const equalActive =
+    nextProps.touchState !== State.UNDETERMINED ||
+    prevProps.isActive === nextProps.isActive
```

Comment:
Need an explanation either through variable name or through comments.

The reason is that because of the `useEffect` function in harp-cell which sets the `touchState` back to `UNDEFINED` whenever it isn't already `UNDEFINED` (implying that it's been pre-toggled) and the `isActive` of the cell changes. Because of this relationship, we can effectively consider an update to `UNDEFINED` as a declaration to rerender, and we will know that the `isActive` is in it's most up to date state and we will use that to determine the style qualities of the cell.

One approach to making this clearer could be to acknowledge the `isTouched` value as part of the `equalActive` calculation more completely. Perhaps by renaming it to `equalActivity` to differentiate it from the input.

```
const equalActivity =
  (prevProps.touchState === nextProps.touchState && nextProps.touchState !== State.UNDETERMINED) ||
  (prevProps.isActive === nextProps.isActive && nextProps.touchState === State.UNDETERMINED)
```

Or something similar..

Reply:
Done